raven/src/main/java/com/getsentry/raven/marshaller/json/JsonMarshaller.java

```diff
@@ -294,10 +294,10 @@ private void writeBreadcumbs(JsonGenerator generator, List<Breadcrumb> breadcrum
             generator.writeNumberField("timestamp", breadcrumb.getTimestamp().getTime() / 1000);
 
             if (breadcrumb.getType() != null) {
-                generator.writeStringField("type", breadcrumb.getType());
+                generator.writeStringField("type", breadcrumb.getType().name().toLowerCase());
```

Comment:
This seems a little magical rather than just maintaining a mapping from enum to string value, IMO.